packages/tsc/index.ts

```diff
@@ -65,7 +58,7 @@ export function run() {
 	}
 }
 
-const removeEmitGlobalTypesRegexp = /[^\n]*__VLS_globalTypesStart[\w\W]*__VLS_globalTypesEnd[^\n]*\n/g;
+const removeEmitGlobalTypesRegexp = /^[^\n]*__VLS_globalTypesStart[\w\W]*__VLS_globalTypesEnd[^\n]*\n?$/mg;
```

Comment:
I think those initial/trailing `\n` make no sense with `m` flag. Those should be ignored anyway when matching with `m` flag.

```suggestion
const removeEmitGlobalTypesRegexp = /^__VLS_globalTypesStart[\w\W]*__VLS_globalTypesEnd.+$/mg;
```

Reply:
cc @blake-newman